codebuild/bin/s2n_open_fds_test.py

```diff
@@ -0,0 +1,81 @@
+import os
+import re
+import sys
+
+# Allow stdin, stdout, stderr, and the file descriptor for dynamic analysis log to remain open at exit
+ACCEPTABLE_OPEN_FDS = 4
+ERROR_EXIT_CODE = 123
```

Comment:
Let's document what this is the exit code for.

Reply:
Upon further reflection, is there a reason to match the return code?

This is returned by the Valgrind process to try and make it clear that this is a "funky" exit.

If it was an exit code of 1 or 2, then it might be a system level error but 123 is a much more conspicuous number. I don't think those motivations apply here, and it would probably be simpler to just return 1 or 0.